apps/evm/src/hooks/useOperationModal/Modal/SupplyForm/index.tsx

```diff
@@ -434,6 +445,26 @@ const SupplyForm: React.FC<SupplyFormProps> = ({ asset, pool, onCloseModal }) =>
     [isWrapUnwrapNativeTokenEnabled, asset.vToken.underlyingToken.tokenWrapped],
   );
 
+  const nativeTokenBalanceTokens = useMemo(() => {
+    return nativeTokenBalanceData
+      ? convertMantissaToTokens({
+          token: asset.vToken.underlyingToken.tokenWrapped,
+          value: nativeTokenBalanceData?.balanceMantissa,
+        })
+      : undefined;
+  }, [asset.vToken.underlyingToken.tokenWrapped, nativeTokenBalanceData]);
+
+  const shouldSelectNativeToken =
+    canWrapNativeToken && nativeTokenBalanceTokens?.gte(asset.userWalletBalanceTokens);
```

Comment:
Same comment as on the Repay form: I wouldn't change the token selected by default.